daemon/internal/cloud_client/change_active_traces_event.go

```diff
@@ -0,0 +1,77 @@
+package cloud_client
+
+import (
+	"github.com/akitasoftware/akita-cli/printer"
+	"github.com/akitasoftware/akita-libs/akid"
+	"github.com/akitasoftware/akita-libs/daemon"
+)
+
+// Indicates that the set of active traces for a service has changed. This is
+// added to the cloud client's main event channel when we get responses to
+// longPollServiceEvent.
+type changedActiveTracesEvent struct {
+	// The service for which the set of active traces has changed.
+	serviceID akid.ServiceID
+
+	// Specifies which traces have been activated, and which have been
+	// deactivated.
+	activeTraceDiff daemon.ActiveTraceDiff
+}
+
+func newChangeActiveTracesEvent(serviceID akid.ServiceID, activeTraceDiff daemon.ActiveTraceDiff) changedActiveTracesEvent {
+	return changedActiveTracesEvent{
+		serviceID:       serviceID,
+		activeTraceDiff: activeTraceDiff,
```

Comment:
It feels sort of like activeTraceDiff should be passed around by pointer here (and elsewhere) instead of copied, but since it's just a collection of slices it's probably fine?

Reply:
I'm explicitly copying it to avoid races. The safety we get seems worth the cost.